Cargo.toml

```diff
@@ -6,5 +6,5 @@ members = ["rust/operator-binary", "rust/csi-grpc"]
 h2 = { git = "https://github.com/stackabletech/h2.git", branch = "feature/grpc-uds" }
 
 [patch."https://github.com/stackabletech/operator-rs.git"]
-# stackable-operator = { git = "https://github.com/stackabletech//operator-rs.git", branch = "feature/listener-implied-labels" }
+stackable-operator = { git = "https://github.com/stackabletech//operator-rs.git", branch = "feat/listener-service-external-traffic-policy" }
```

Comment:
Reminder

Reply:
@sbernauer This is unblocked now that we have https://github.com/stackabletech/operator-rs/pull/773 right?

Reply:
Oh yes, I will pick this PR up again (feel free to push and merge yourself if I'm too slow)

Reply:
I'm not in a rush here, just ran into the PR by accident. ^^